keras_cv/models/object_detection/retinanet/retinanet_test.py

```diff
@@ -234,21 +234,63 @@ def test_saved_model(self, save_format, filename):
 
 @pytest.mark.large
 class RetinaNetSmokeTest(tf.test.TestCase):
-    def test_backbone_preset_weight_loading(self):
-        # Check that backbone preset weights loaded correctly
-        # TODO(lukewood): need to forward pass test once proper weights are
-        # implemented
-        keras_cv.models.RetinaNet.from_preset(
-            "resnet50_v2_imagenet",
-            num_classes=20,
-            bounding_box_format="xywh",
-        )
+    def test_backbone_preset(self):
```

Comment:
Sorry for the late response, but this should be using the `presets` and `presets_with_weights` properties to grab the presets. We do not want to enumerate these manually.

Reply:
Yes, also we want the same test for YOLOv8 @ianstenbit

Reply:
Yeah, I think @IMvision12 mentioned that a follow-up PR for YOLOV8 is en route

Reply:
@jbischof Yeah i will create a follow-up PR for yolov8 and for retinanet once #1858 is approved :)